server/src/main/java/org/elasticsearch/index/codec/postings/Lucene90BlockTreeTermsWriter.java

```diff
@@ -0,0 +1,1191 @@
+/*
```

Comment:
Are we forking this from Lucene? Why so?

Reply:
Unfortunately we have to *copy* (not fork per se). Since the writer has been moved to backward compat test in Lucene - so is not in any shipping Lucene jar :-(

Reply:
I see, thanks for explaining. That is odd!